ballerina/api.bal

```diff
@@ -32,7 +35,7 @@ public isolated function callLlm(Prompt prompt, Model model = getDefaultModel(),
 } external;
 
 public const annotation LlmCall on source external;
-
+public annotation Schema schemaAnnot on type;
```

Comment:
These don't have to be public, right?

Also have the type and the annotation together.

Reply:
We needs to have a public type
Other wise we cannot use this in a code modifier